### PR TITLE
Add Relay blog post

### PR DESCRIPTION
**Pull request by wincent**
This is a blog post providing some additional info about Relay and GraphQL.

I've circulated it among the team for technical feedback, but it's the first time I've prepped one of these so let me know if I've done anything wrong.

### CODE REVIEW

**Comment by SanderSpies** [comment]
From what I've heard so far, it's that people think that Relay is really specific to the Facebook scenario and not interesting for everyone. Also how the server-side might be implemented gives people, including me, a lot of question marks (I guess mentioning spec already clarifies it a bit though :-)). So it might be interesting to tell a bit more about those two subjects.

File: docs/_posts/2015-02-17-introducing-relay-and-graphql.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+The two projects &mdash; Relay and GraphQL &mdash; have been in use in production at Facebook for some time, and we're excited to be bringing them to the world as open source in the future. In the meantime, we wanted to share some additional information about the projects here.
+
+<script async class="speakerdeck-embed" data-id="7af7c2f33bf9451a892dcd91de55b7c2" data-ratio="1.29456384323641" src="//speakerdeck.com/assets/embed.js"></script>
+
+## What is Relay?
+
+Relay is a new framework from Facebook that provides data-fetching functionality for React applications. It was announced at React.js Conf (January 2015).
+
+Each component specifies its own data dependencies declaratively using a query language called GraphQL. The data are made available to the component via properties on `this.props`.
```
Comment (zpao) [inline docs/_posts/2015-02-17-introducing-relay-and-graphql.md]: I've gotten flak for it before, but I like "data" as a mass noun using singular verbs

> The data *is* made available

I don't care that much, just one of those things…

File: docs/_posts/2015-02-17-introducing-relay-and-graphql.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Open source plans
+
+We're working very hard right now on getting both GraphQL (a spec, and a reference implementation) and Relay ready for public release (no specific dates yet, but we are super excited about getting these out there).
+
+In the meantime, we'll be providing more and more information in the form of blog posts (and in [other channels](https://gist.github.com/wincent/598fa75e22bdfa44cf47)). As we get closer to open source release, you can expect more concrete details, syntax and API descriptions and more.
```
Comment (zpao) [inline docs/_posts/2015-02-17-introducing-relay-and-graphql.md]: > get closer to open source release

get closer to **the** open source release

File: docs/_posts/2015-02-17-introducing-relay-and-graphql.md (hunk 1 above)
Comment (wincent) [inline docs/_posts/2015-02-17-introducing-relay-and-graphql.md]: Will update.

File: docs/_posts/2015-02-17-introducing-relay-and-graphql.md (hunk 2 above)
Comment (wincent) [inline docs/_posts/2015-02-17-introducing-relay-and-graphql.md]: Will update.

**Comment by zpao** [comment]
:thumbsup: I'll update the filename and get this in tonight, then we can rebuild the site in the morning (better timing for more of the internet than 9PM <abbr title="Yes, technically it's PDT">PST</abbr>)

**Pull request closed without merging**